index.html

```diff
@@ -8,41 +8,89 @@
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
 <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
 <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;800&display=swap" rel="stylesheet">
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-  <header>
+  <header class="header-m">
     <div class="logo">
       <img  src="./images/image_geometry_1.png" alt="logo image">
    </div>
     <nav>
       <div class="menu">
            <i class="fa fa-bars"></i>
       </div>
+        <ul class="menu-d-list">
+          <li>Hello <hr></li>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      
     </nav>
   </header>
   <main>
   
     <section class="headline"> 
+      <div class="headline-d">
+
         <h1 class="myname">Charlie 
         Giorgania
         </h1> 
-        <p  class="headline-p">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br> <br>
+
+        <p class="headline-p">Hello! I am a software developer! I can help you build a product, 
+          feature or website. Take a look at my work. <br> <br>
   
           If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-      <button class="headline-btn">
-        Start collaboration
-      </button>
+
+        <p class="headline-p-d">Hello! I am a software developer! I can help you build a product, 
+          feature or website. Take a look at my work. <br> <br>
+  
+          If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <button class="headline-btn">
+          Start collaboration
+        </button>
+
+      </div>
+
       <div class="headline-img">
-       
+        
       </div>
-  </section>
- <section class="projects">
+      <div class="image-d">
+        <img class="img1-d"src="./images/image_dots.png" alt="dotd image">
+
+        <img class="img2-d" src="./images/image-geometry_1.png" alt="introimg">
+    </div>
+
+    </section>
+  <section class="projects">
+        <div class="project-d-header">
           <h2 class="project-title">Projects</h2>
+          
+          <div class="project-card-d">
+              <div class= "img-d"><img class= "img3-d" src="./images/image_dots.png" alt="dotd image"></div>
+              <div class="project1-card-d">  
+                  <div class="project1-img-d">
+                  </div>
+                  <div class="project-des-d">
+                      <h2 class="sub-title-d">Project name goes here</h2>
```

Comment:
Great job on the projects section and the project card titles! 👍 However, I noticed that you used the `<h2>` tag for both the projects section title and the individual project card titles. While the visual presentation may be consistent, it is important to consider semantic HTML and accessibility concerns.

Semantic HTML helps provide structure and meaning to your content, allowing assistive technologies and search engines to understand the hierarchy and purpose of your headings. To ensure proper semantic structure, it is recommended to use different heading levels for different sections of your webpage.

In this case, I suggest using a higher-level heading tag, such as `<h2>` for the projects section title, and a lower-level heading tag, such as `<h3>` for the project card titles. This way, you maintain a clear hierarchy and improve the accessibility of your webpage.

For more information on heading elements and accessibility concerns, I recommend checking out the article [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns).

By following these best practices, you can ensure your code aligns with semantic HTML standards and provides a more accessible experience for all users. Keep up the great work! 😄